app/views/content/funding-and-support/scholarships-and-bursaries.md

```diff
@@ -71,60 +39,68 @@ To be eligible for a bursary, you’ll need at least one of the following:
 
 If your course combines 2 subjects (for example, English and history), at least 50% of your course content must be made up of the bursary subject to be eligible.
 
-You’ll also need to meet the bursary scheme’s individual terms and conditions.
+#### How bursaries are paid
 
-[Find out more about your eligibility for teaching bursaries](https://www.gov.uk/government/publications/funding-initial-teacher-training-itt/funding-initial-teacher-training-itt-academic-year-2024-to-2025).
+A bursary is paid over a minimum of 10 months by your teacher training provider. They will confirm when you will get the payments. 
 
-### Scholarships
+You will not need to pay tax or National Insurance on bursary payments. 
 
-#### Eligibility
+#### How to get a bursary
+
+You do not need to apply for a bursary. When you apply for teacher training, your provider will check if you're eligible for a bursary. If you are, your provider will confirm this in writing before you start your course. 
+
+## Eligibility for scholarships
 
 To be eligible for a scholarship, you’ll need at least one of the following:
 
 * a bachelor’s degree class 2:1 (honours) or higher
 * a master’s degree
 * a PhD
 
-In exceptional circumstances, if you achieved a 2:2 in your degree you may still be awarded a scholarship if you have significant relevant experience.
+If you achieved a 2:2 in your degree, you may still be eligible. You would need to show you have enough relevant experience and subject knowledge. 
+
+As scholarships are awarded by independent organisations, they set their own assessment requirements and application deadlines.
 
-#### Benefits
+They offer more than just financial benefits. For example, they can provide professional body membership, subject-specific support, mentoring, events and classroom resources. 
 
-Scholarships are provided by charitable organisations with a passion for their subject area.
+#### How scholarships are paid
 
-They offer more than just financial benefits, including:
+A scholarship is paid over a minimum of 10 months by your teacher training provider. They will confirm when you will get the payments. 
 
-* subject specialist events and webinars
-* online and face-to-face workshops
-* mentoring and coaching
-* a community of ex-scholars, teachers and subject leaders
-* access to classroom resources
-* networking events during your teacher training and beyond
+You will not need to pay tax or National Insurance on scholarship payments. 
 
-#### How to apply
+#### How to apply for a scholarship
```

Comment:
As in previous comment, I think it is worth saying something about application deadlines in this section

For reference, in production we say `Each scholarship body will have its own deadline for applications. You can find these on their individual websites.`

(we don't have to repeat this)